Library/ViewModels/LoginToutViewModelTests.swift

```diff
@@ -585,4 +590,26 @@ final class LoginToutViewModelTests: TestCase {
 
     self.dismissViewController.assertValueCount(1)
   }
+
+  @available(iOS 13, *)
+  func testDidSignInWithApple() {
```

Comment:
Can you add a test for the `prepareSignInWithAppleRequest` output?

Reply:
Goot call